src/Compilers/CSharp/Test/Emit/Attributes/InternalsVisibleToAndStrongNameTests.cs

```diff
@@ -1739,7 +1739,7 @@ public class C
             }
         }
 
-        [Fact]
+        [ConditionalFact(typeof(IsEnglishLocal))]
```

Comment:
This one shouldn't need to be conditional.  I don't see any hard-coded strings that would affect it.